clarifai/utils/logging.py

```diff
@@ -151,3 +201,153 @@ def display_concept_relations_tree(relations_dict: Dict[str, Any]) -> None:
     for child in children:
       tree.add(child)
     rprint(tree)
+
+
+def _default_json_default(obj):
+  """
+  Handle objects that could not be serialized to JSON automatically.
+
+  Coerce everything to strings.
+  All objects representing time get output as ISO8601.
```

Comment:
this in UTC?

Reply:
yeah i think so. `2024-09-24T21:31:15.411793Z`